{
	"id": "ALAS2-2023-2317",
	"advisories": [
		{
			"id": "ALAS2-2023-2317",
			"title": "Amazon Linux 2 2017.12 - ALAS2-2023-2317: important priority package update for python3",
			"description": "Package updates are available for Amazon Linux 2 that fix the following vulnerabilities:\nCVE-2022-48565:\n\tAn XML External Entity (XXE) issue was discovered in Python through 3.9.1. The plistlib module no longer accepts entity declarations in XML plist files to avoid XML vulnerabilities.\n\nCVE-2020-27619:\n\tIn Python3's Lib/test/multibytecodec_support.py CJK codec tests call eval() on content retrieved via HTTP.\n\nCVE-2020-26116:\n\tA flaw was found in Python. The built-in modules httplib and http.client (included in Python 2 and Python 3, respectively) do not properly validate CRLF sequences in the HTTP request method, potentially allowing manipulation to the request by injecting additional HTTP headers. The highest threat from this vulnerability is to confidentiality and integrity.\n",
			"severity": [
				{
					"type": "vendor",
					"source": "linux-security@amazon.com",
					"vendor": "important"
				}
			],
			"references": [
				{
					"source": "linux-security@amazon.com",
					"url": "https://alas.aws.amazon.com/AL2/ALAS-2023-2317.html"
				},
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-26116"
				},
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-27619"
				},
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-48565"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2020-26116",
			"references": [
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-26116"
				}
			]
		},
		{
			"id": "CVE-2020-27619",
			"references": [
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-27619"
				}
			]
		},
		{
			"id": "CVE-2022-48565",
			"references": [
				{
					"source": "linux-security@amazon.com",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-48565"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-tools",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-tkinter",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-test",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-debug",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-debuginfo",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-libs",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		},
		{
			"ecosystem": "amazon:2",
			"vulnerable": true,
			"package": {
				"name": "python3-devel",
				"repositories": [
					"amzn2-core"
				],
				"architectures": [
					"i686"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.7.10-1.amzn2.0.1"
					}
				],
				"fixed": [
					"0:3.7.10-1.amzn2.0.1"
				]
			}
		}
	],
	"data_source": "amazon"
}
